nlp/model/sagemaker/src/main/kotlin/SagemakerEngineProvider.kt

```diff
@@ -56,4 +49,13 @@ class SagemakerEngineProvider : NlpEngineProvider {
         // do not tokenize anything at this stage
         override fun tokenize(context: TokenizerContext, text: String): Array<String> = arrayOf(text)
     }
+
+    override fun healthcheck(): () -> NlpHealthcheckResult = {
+        val grouped = SagemakerClientProvider.getAllClient().groupBy { it.name }.withDefault { emptyList() }
```

Comment:
Actually, the `name` property could be replaced with a `type` that uses an enum instead of a string